src/texts.js

```diff
@@ -0,0 +1 @@
+export const OPEN_DRAWER_LABEL = 'Open drawer'
```

Comment:
`'Open Drawer'`. Do we need to translate this?

Reply:
I'll change the capitalization. This text is used as default label for the drawer open button if no other text is submitted.

Reply:
I think it's fine for now.